src/app/business/block/bucket-detail/bucket-detail.component.ts

```diff
@@ -260,7 +260,7 @@ export class BucketDetailComponent implements OnInit {
               return folderContain;
             })
             this.allDir.forEach(val=>{
-              val.ObjectKey = val.ObjectKey.slice(this.folderId.length);
+              val.Key = val.Key.slice(this.folderId.length);
```

Comment:
This is fixed in #259. It will be merged before this PR. Please remove this from here.

Reply:
> This is fixed in #259. It will be merged before this PR. Please remove this from here.

Ok, I've changed it